sdk/appconfiguration/azure-data-appconfiguration/README.md

```diff
@@ -17,6 +17,37 @@ Use the client library for App Configuration to create and manage application co
 - [App Configuration Store][app_config_store]
 
 ### Include the Package
+#### Include the BOM file
+
+Please include the azure-sdk-bom to your project to take dependency on the latest GA version of the library.
+
+```xml
+<dependencyManagement>
+    <dependencies>
+        <dependency>
+            <groupId>com.azure</groupId>
+            <artifactId>azure-sdk-bom</artifactId>
+            <version>1.0.3</version>
+            <type>pom</type>
+            <scope>import</scope>
+        </dependency>
+    </dependencies>
+</dependencyManagement>
+```
+and then include the direct dependency in the dependencies section without the version tag.
```

Comment:
Let's make this a sentence on its own rather than a continuation from before the code block

```suggestion
Then include the direct dependency in the dependencies section without the version tag.
```

Reply:
Based on my other comment above we should more clearly indicate that the `dependencies` block that the direct dependency is included in is not within the `dependencyManagement` block but a completely separate block.

```xml
<dependencyManagement>
    <dependencies>
        <dependency>
            <groupId>com.azure</groupId>
            <artifactId>azure-sdk-bom</artifactId>
            <version>1.0.3</version>
            <type>pom</type>
            <scope>import</scope>
        </dependency>
    </dependencies>
</dependencyManagement>

<dependencies>
    <dependency>
        <groupId>com.azure</groupId>
        <artifactId>azure-data-appconfiguration</artifactId>
    </dependency>
</dependencies>
```